### ISSUE TITLE
Normalize() transform has undocumented side effect

### ISSUE THREAD

**Issue by keunhong** [pytorch/vision#517]
It seems like [F.normalize()](https://github.com/pytorch/vision/blob/master/torchvision/transforms/functional.py#L168) does all of its operations in-place. 

I feel like this should either be very clearly documented or be hidden behind a flag.

This really bit me when I was just sending some batch images to Visdom but my denormalization function was silently denormalizing my batch before doing a forward pass on the network. It was hidden to me until now and surfaces because a copy disappeared when I removed all Variable() wrappers.
